lms/static/scripts/frontend_apps/components/ErrorDisplay.tsx

```diff
@@ -69,14 +70,15 @@ function supportURL(errorMessage: string, error: ErrorLike): string {
   );
 
   const details = formatErrorDetails(error);
-  if (error.errorCode || details) {
+  if (error.errorCode || details || debug) {
     const content = `
 ----------------------
 Feel free to add additional details above about the problem you are experiencing.
 The error information below helps our team pinpoint the issue faster.
 ----------------------
 Error code: ${error.errorCode ?? 'N/A'}
 Details: ${formatErrorDetails(error) || 'N/A'}
+Debug: ${JSON.stringify(debug) || 'N/A'}
```

Comment:
Instead of adding a "big" blob of JSON to the URL, we could maybe just pick the individual values we need.

This should work nonetheless.

Reply:
It's a tricky balance. I reckon  the drawbacks of this approach is that it gets to big but allows the same approach to work in contexts where the debug info might be different. 

For example we could include here other parameters when it's an unauthorized launch (ie we don't know AI, course, user or assignment).

I don't think this could grow that much as it does provide a good context right now but I'm bias as this is so much simpler for be to deal in the frontend change.

Reply:
I think we can go this way for now and change it later if it presents some problem.